cvat-ui/src/components/file-manager/remote-browser.tsx

```diff
@@ -171,6 +171,18 @@ function RemoteBrowser(props: Props): JSX.Element {
             }
         };
         collectNodes(content);
+
+        if (manifestPath) {
```

Comment:
Is this manifest file labeled in screenshot going to be added to the files being uploaded and is it going to affect to predefined sorting method #6836? @Marishka17 
![image_2023-09-12_13-07-53](https://github.com/opencv/cvat/assets/23011104/61974cc8-be66-4324-8f81-33bb1d9bcf82)

Reply:
@bedilbek, Yes, such manifest will be added to the list of files and the sequence of frames will be determined based on manifest content (in case of using `predefined` sorting).